docs/_tutorials/getting_started.md

```diff
@@ -104,63 +106,60 @@ const app = new App({
-<summary markdown="0">
-<h4>Using a local Request URL for development</h4>
-</summary>
-
-If you’re just getting started with your app's development, you probably don’t have a publicly accessible URL yet. Eventually, you’ll want to set one up, but for now a development proxy like [ngrok](https://ngrok.com/) will create a public URL and tunnel requests to your own development environment. We've written a separate tutorial about [using ngrok with Slack for local development](https://api.slack.com/tutorials/tunneling-with-ngrok) that should help you get everything set up.
-
-Once you’ve installed a development proxy, run it to begin forwarding requests to a specific port (we’re using port `3000` for this example, but if you customized the port used to initialize your app use that port instead):
+1. Head to your app's configuration page (click on the app [from your app management page](https://api.slack.com/apps)). Navigate to **Socket Mode** on the left side menu and toggle to enable. 
 
-```shell
-ngrok http 3000
-```
+2. Go to **Basic Information** and scroll down under the App Token section and click **Generate Token and Scopes** to generate an app token. Add the `connections:write` scope to this token and save the generated `xapp` token, we'll use that in just a moment. 
 
-![Running ngrok](../assets/ngrok.gif "Running ngrok")
+Finally, it's time to tell Slack what events we'd like to listen for. Under **Event Subscriptions**, toggle the switch labeled **Enable Events**. 
 
-The output should show a generated URL that you can use (we recommend the one that starts with `https://`). This URL will be the base of your request URL, in this case `https://8e8ec2d7.ngrok.io`.
-
----
-</details>
-
-Now you have a public-facing URL for your app that tunnels to your local machine. The Request URL that you use in your app configuration is composed of your public-facing URL combined with the URL your app is listening on. By default, Bolt apps listen at `/slack/events` so our full request URL would be `https://8e8ec2d7.ngrok.io/slack/events`.
-
-> ⚙️Bolt uses the `/slack/events` endpoint to listen to all incoming requests (whether shortcuts, events, or interactivity payloads). When configuring endpoints within your app configuration, you'll append `/slack/events` to all request URLs.
-
-Under the **Enable Events** switch in the **Request URL** box, go ahead and paste in your URL. As long as your Bolt app is still running, your URL should become verified.
+When an event occurs, Slack will send your app some information about the event, like the user that triggered it and the channel it occurred in. Your app will process the details and can respond accordingly.
 
-After your request URL is verified, scroll down to **Subscribe to Bot Events**. There are four events related to messages:
+Scroll down to **Subscribe to Bot Events**. There are four events related to messages:
 - `message.channels` listens for messages in public channels that your app is added to
```

Comment:
```suggestion
- [`message.channels`](https://api.slack.com/events/message.channels) listens for messages in public channels that your app is added to
```

Reply:
These links aren't related to your PR, but I just noticed them and felt they may be helpful to link out to.